code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -161,6 +161,7 @@ obj/item/weapon/robot_module/proc/fix_modules() //call this proc to enable click
 	src.modules += new /obj/item/weapon/surgicaldrill(src)
 	src.modules += new /obj/item/weapon/revivalprod(src)
 	src.modules += new /obj/item/weapon/crowbar(src)
+	src.modules += new /obj/item/weapon/inflatable_dispenser/robot
```

Comment:
I don't agree but I don't see this as major enough to change right away.  We can always go back and  worry about the beepbeeppowercreep afterwards.